etna/records/views/images.py

```diff
@@ -79,10 +66,8 @@ def image_browse(request, iaid):
     if not page.is_digitised:
         raise Http404
```

Comment:
Why Http404 always ? I think we should check the response status code from Kong first, and if the status is 404 then maybe then.

Reply:
A 404 is an appropriate response if the resource (not necessarily a file) we're requesting can't be found:

https://docs.djangoproject.com/en/4.0/topics/http/views/#the-http404-exception

In this instance, we're saying if the page doesn't have images (if it's not digitised) then 404.